src/views/guidelines/guidelines.jsx

```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+var FormattedMessage = require('react-intl').FormattedMessage;
+
+var render = require('../../lib/render.jsx');
+var Page = require('../../components/page/page.jsx');
+var Box = require('../../components/box/box.jsx');
+
+require('./guidelines.scss');
+
+var Guidelines = React.createClass({
+    type: 'Guidelines',
+    render: function () {
+        return (
+            <Box title={<FormattedMessage id='guidelines.title' />}>
+                <div className="bottom">
```

Comment:
I think that this div inside `Box` is not necessary. I think it'd be better to remove this, and make the header and unordered list direct children of the `Box` element:

```
<Box>
    <p>...</p>
    <ul>...</ul>
</Box>
```
Then you can remove the styling for `.bottom` from the scss file too